src/components/cardHeader/index.js

```diff
@@ -1,18 +1,15 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-function ProfileHeader({userDetails}) {
+const ProfileHeader = ({ userDetails }) => {
   const { lastSeen, name, picture } = userDetails;
```

Comment:
use || {} while extracting value from de-structured value. it may be falsy value.